server/src/utils/logger.util.ts

```
@@ -0,0 +1,37 @@
+import { createLogger, format, transports } from 'winston';
+import path from 'path';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)